src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -183,6 +215,13 @@ private void configChangeHandler(WhatHappened whatHappened, Node node) {
     protected void startup() throws InterruptedException {
         context.get(CertificateManager.class).startMonitors();
         super.startup();
+
+        synchronized (configLock) {
+            if (configurationErrored) {
+                configurationErrored = false;
+                onConfigurationChanged();
+            }
+        }
```

Comment:
I thought the issue was that if we are broken, then Nucleus _doesn't_ call `startup`?